test/tests/problems/no_solve/tests

```diff
@@ -22,9 +22,8 @@
     type = 'RunApp'
     prereq = eigen_problem_solve_fail
     requirement = 'The system shall have the ability to disable the nonlinear-eigen system solve in a simulation.'
-    # regex meant to make sure no messaged about solve failing is printed last
-    # \S+ is to consume the ansi color characters
-    expect_out = 'Solve Skipped!\S+\s+(?!Aborting)'
+    # regex meant to make sure no message about solve failing is printed
+    expect_out = 'Solve Skipped!(?![\s\S]*Aborting)'
```

Comment:
```suggestion
    expect_out = 'Solve Skipped!(?!.*Aborting)'
```

Reply:
I think there's some value in trying to make the regex work independently of the options we pass to re.search in case someone wants to try and change them in the future